build.sbt

```diff
@@ -21,7 +21,8 @@ val collections =
       name := "collection-strawman",
       libraryDependencies ++= Seq(
         "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-        "com.novocode" % "junit-interface" % "0.11" % Test
+        "com.novocode" % "junit-interface" % "0.11" % Test,
+        "org.scalatest" %% "scalatest" % "3.0.1" % Test
```

Comment:
Indeed.  I'm not sure that there's much value-add for that above what collections-laws should already do (explicitly test relationships that should always hold, with fuzzing of data but not operations), but if it's not too hard to manage it's probably a good idea before we make the switch.